src/main/java/com/wwi21sebgroup5/cinema/entities/Role.java

```diff
@@ -1,4 +1,5 @@
 package com.wwi21sebgroup5.cinema.entities;
 
-public class Role {
+public enum Role {
```

Comment:
3. rolle Mitarbeiter? Oder erst nach dem Walking Skeleton?

Reply:
Können wir gerne in nem neuen Issue mit reinbringen, davor sollten wir uns aber mal hinsetzen und die Rollen genauer erstellen mit den Bereichen, die wir limitieren wollen.